app/pages/NavigatorDashboard/NavigatorDashboard.tsx

```diff
@@ -1,19 +1,64 @@
 import React from "react";
+import qs from "qs";
+import { useHistory } from "react-router-dom";
+
+import { icon } from "assets";
+import { Button } from "components/ui/inline/Button/Button";
+
 import styles from "./NavigatorDashboard.module.scss";
 
+const AdvancedSearch = () => {
+  const history = useHistory();
+  let searchValue = "";
+  const submitSearch = () => {
+    if (searchValue) {
+      const query = qs.stringify({ query: searchValue });
+      history.push(`/search?${query}`);
+    }
+  };
+
+  return (
+    <div className={styles.advancedSearchContainer}>
+      <h1 className={styles.header}>Hi, Navigator :)</h1>
+      <p className={styles.blurb}>
+        Please use the search bar to find resources for the individuals
+        you&apos;re assisting. You can enter keywords such as
+        &ldquo;shelter,&ldquo; &ldquo;food assistance,&ldquo; or
+        &ldquo;employment support,&ldquo; to discover relevant programs and
+        services.
+      </p>
+      <form
+        className={styles.searchBoxForm}
+        onSubmit={(evt) => {
+          evt.preventDefault();
+          submitSearch();
+        }}
+      >
+        <div className={styles.searchInputContainer}>
+          <input
+            type="text"
+            placeholder="Emergency Shelter"
+            className={styles.searchInput}
+            onChange={(evt) => {
+              searchValue = evt.target.value;
```

Comment:
This seems a little bit dangerous, since you're mutating a non-state, non-ref variable, which means that React doesn't know about it and it's not part of the usual way you manage state outside of React. Not a big deal to leave it here, if you were planning to completely revamp this in the next PR, but in case you weren't planning to change this, I think it'd be a lot safer to either make `searchValue` a regular React state variable, or to remove the `onChange` prop completely to leave this as an uncontrolled form component.

Reply:
Thank you! I will merge this i and fix this in my subsequent https://github.com/ShelterTechSF/askdarcel-web/pull/1345 PR